src/Views/gagepage.html

```diff
@@ -59,6 +59,10 @@ <h4 class="modal-title">StreamStats Data-Collection Station Report</h4>
                                 <th scope="row">Agency</th>
                                 <td>{{vm.gage.agency.name}}</td>
                             </tr>
+                            <tr>
+                               <th scope="row">NWIS Period of Record</th>
+                               <td>{{vm.gage.StartDate | date:'MM/dd/yyyy'}} - {{vm.gage.EndDate | date:'MM/dd/yyyy'}}</td>
```

Comment:
Show "Undefined" instead of "-" if there is no valid StartDate or EndDate

![image](https://user-images.githubusercontent.com/40237491/159062704-6336d78e-faec-4b08-8779-d37e83e61231.png)

Reply:
Resolved in https://github.com/USGS-WiM/StreamStats/pull/1237/commits/5c3a49d9c5218fa91e08b702a0f06a0a7b904816